implementations/micrometer-registry-elastic/src/test/java/io/micrometer/elastic/AbstractElasticsearchMeterRegistryIntegrationTest.java

```diff
@@ -40,9 +40,9 @@
 @Tag("docker")
 abstract class AbstractElasticsearchMeterRegistryIntegrationTest {
 
-    protected static final String VERSION_7 = "7.17.8";
+    protected static final String VERSION_7 = "7.7.1";
```

Comment:
If we want a specific minor version of 7 for testing purposes, I think it'd be best to leave a comment so this doesn't get overwritten.